# recover
不是所有的异常都能被捕获到的，像fatal error 和runtime.throw 都是不能被recover的